lib/anycablebility/client.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'thread'
+require 'json'
+require 'websocket-eventmachine-client'
+
+require 'anycablebility/waiter'
+
+module Anycablebility
+  # This is a simple ActionCable client created for testing purposes
+  class Client
+    attr_reader :state
+
+    def initialize(cable_url, debug = false, ignore_message_types = [])
```

Comment:
Agree. Command line arguments would be enough.

Let's introduce a _global_ Logger instead (say, `Anycablebility.logger`). Or we can even use `Anycable.logger`, btw.